src/test/java/teammates/test/cases/util/StringHelperTest.java

```diff
@@ -127,6 +128,33 @@ public void testKeyEncryption() throws InvalidParametersException {
         assertEquals(msg, decrptedMsg);
     }
 
+    private static String encryptWithoutSpecifyingAlgorithmParams(String value) throws Exception {
+
+        SecretKeySpec sks = new SecretKeySpec(StringHelper.hexStringToByteArray(Config.ENCRYPTION_KEY), "AES");
+        Cipher cipher = Cipher.getInstance("AES");
+        cipher.init(Cipher.ENCRYPT_MODE, sks, cipher.getParameters());
+        byte[] encrypted = cipher.doFinal(value.getBytes());
+        return StringHelper.byteArrayToHexString(encrypted);
+
+    }
+
+    @Test
+    public void testDefaultAesCipherParams() throws Exception {
+        String plaintextLength28 = "AAAAAAA";
+        //Using plaintext > 128 bits to check any differences operations.
+        String plaintextLength128 = "AAAAAAAAAABBBBBBBBBBCCCCCCCCCCCC";
+        String plaintextLength136 = plaintextLength128 + "AA";
+        String expectedCiphertext;
+        String actualCiphertext;
+        String[] inputMsgs = {plaintextLength28, plaintextLength128, plaintextLength136};
+        for (String msg : inputMsgs) {
+
+            actualCiphertext = encryptWithoutSpecifyingAlgorithmParams(msg);
+            expectedCiphertext = StringHelper.encrypt(msg);
+            assertEquals(expectedCiphertext, actualCiphertext);
+        }
```

Comment:
This whole method has three test cases. It can be written like this:
```java
@Test
public void testDefaultAesCipherParams() throws Exception {
    assertAbc("AAAAAAAAAABBBBBBBBBBCCCCCCCCCCCC") // Short explanation of the case
    assertAbc("AAAA") // Short explanation of the case
    assertAbc("AAAA") // Short explanation of the case
}
```

Of course you should give a better name than `assertAbs` and the test cases in my example are dummies.